roles/sft-server/vars/main.yml

```diff
@@ -0,0 +1,30 @@
+# NOTE: considered to be constants due to high precedence
+
+SFT_ARTIFACT_TARGET_PATH: '/tmp/sft-artifact.tar.gz'
```

Comment:
It is odd practice to use variable names in ALL_UPPERCASE for ansible; I've never seen an ansible role or playbook using uppercase for its variables. Could you use lowercase variables?

Reply:
Variables that are supposed to be (or available for being) overridden are defined in `defaults/main.yml`. The ones that are not, are defined in `vars/main.yml`. As you said, due to Ansible's precedence, it is very *hard* to override the latter. One might say *immutable* even, like constants. Typically, these kind of immutable variables are written in all uppercase. This way, it is easier to distinguish between those that can be overridden and those that are cannot when reading through the code.

Does this practice make sense to you? Should I keep it or change it as per your request?

Reply:
Using uppercase for "unchangeable constants" is not a practice that anyone writing ansible does (at least not that I have seen). Searching for style guides, you can find e.g. https://github.com/whitecloud/ansible-styleguide#variable-names explicitly saying ALL_UPPERCASE is bad. None of our current ansible code thus far uses ALL_UPPERCASE anywhere; let's please not start with this style.
 
It may be a convention in java and C code, but it's not in ansible, let's please not deviate from current standards.

Reply:
Adjusted.